index.js

```diff
@@ -107,14 +107,20 @@ function channels(funcs, context) {
 }
 
 function app(elem, observ, render, opts) {
+    if (!elem) {
+        throw new ReferenceError(
```

Comment:
Let's not use reference error as that is meant for the language. We can just use Error.